lecture/forms.py

```diff
@@ -0,0 +1,8 @@
+from django.forms import ChoiceField
+from django.forms import Form
+
+from le_dash.rollcall import episodes_by_series
+
+
+class MpidForm(Form):
+    mpid = ChoiceField(label='media package id', choices=episodes_by_series)
```

Comment:
it's cool that you went with a dropdown but it introduces the issue that without additional filtering (term, year, series) it's not practical to show every possible option. Did you have thoughts about that?